src/filters/detector.cpp

```diff
@@ -6,10 +6,14 @@ namespace mesos {
 namespace serenity {
 
 Try<Nothing> DetectorFilter::consume(const ResourceUsage& in) {
```

Comment:
I think we could be more precise with the naming of 'detector'. What does it detect? Hard to tell from the name.